crates/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -235,6 +239,9 @@ where
         let toml = remove_packages(&current_manifest_contents, packages.iter().cloned())?;
         // TODO: enable transactions once build is re-implemented
         // self.transact_with_manifest_contents(toml.to_string(), nix, system).await?;
+        debug!("writing new manifest to {:?}", self.manifest_path());
+        std::fs::write(self.manifest_path(), toml.to_string())
+            .map_err(EnvironmentError2::UpdateManifest)?;
```

Comment:
nit: should this conditionally write the manifest like install depending on if updates were made?

Reply:
I dont think we get to writing _nothing_, if nothing was removed we will get an error earlier from `remove_packages`.